src/startup.rs

```diff
@@ -0,0 +1,27 @@
+use log::LevelFilter;
+
+/// Contains a collection of functions that are called at the start of the program.
```

Comment:
```suggestion
//! Contains a collection of functions that are called at the start of the program.

use log::LevelFilter;
```
https://doc.rust-lang.org/rustdoc/index.html#outer-and-inner-documentation